provision.yaml

```diff
@@ -0,0 +1,6 @@
+default:
+  provisioner: vmpooler
+  images: ['win-2012r2-x86_64']
+release_checks:
+  provisioner: vmpooler
+  images: ['win-2012r2-wmf5-x86_64', 'win-2016-x86_64', 'win-2019-core-x86_64']
```

Comment:
@eimlav the reason we chose this set of machines to test against is for two reasons:
1) This is the set of machines that Jenkins tested against
2) Puppetlabs-dsc module requires Powershell 5

Reply:
Any particular reason we don't use the core images for this?

Reply:
nice catch, fixed it